documentation/source/objects/font.rst

```diff
@@ -134,7 +134,7 @@ Interacting with glyphs at the font level is a shortcut for interacting with gly
 
 Does the same thing as::
 
-	>>> glyph = font.getLayer(font.defaultLayer).newGlyph("A")
+	>>> glyph = font.getLayer(font.defaultLayer.name).newGlyph("A")
```

Comment:
Good catch, but this should be `font.defaultLayerName`, either works, but this is more clear.